cmd/definitions/type.go

```diff
@@ -660,6 +661,18 @@ func mergePairs(global, service map[string]*Pair) map[string]*Pair {
 	return ans
 }
 
+func addFeaturePairs(pairs map[string]*Pair, ns specs.Namespace) {
+	for _, feature := range ns.Features {
+		featurePair := &Pair{
+			Name:        "enable_" + feature,
```

Comment:
I prefer `name := fmt.Sprintf("enable_%s", feature)` and use `name` in the following.